### ISSUE TITLE
Improve group page performance

### ISSUE THREAD

**Issue by MrPetovan** [friendica/friendica#8547]
Reported in https://forum.friendi.ca/display/7f2cc12a-205e-a5ea-66c6-e40674705693

Every time a contact is added to a group from a group page, the whole page HTML is generated and fetched asynchronously. Since there is no pagination, the performance cost increases with any additional contact.

On frio, I've been able to cull the contact row template, which would save about half the page size on each Ajax call, but ultimately a better solution is to allow adding multiple contacts to a group at once and then reload the page only once.

**Comment by annando** [comment]
We definitely need a better user interface for this process.

**Comment by AlfredSK** [comment]
Yes. Batch processing would be cool - like we do it with deleting/blocking/archiving contacts.

**Comment by MrPetovan** [comment]
The user admin view is a technical one, not sure it's relevant for user-facing interface.
